<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Submenu</class>
 <widget class="QDialog" name="Submenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configurar SubMenu</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>931</width>
     <height>421</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Itens e Sub-Menus</string>
    </attribute>
    <widget class="QTableView" name="tableView_itens">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>641</width>
       <height>321</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_menu">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_filtrar">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>30</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Filtrar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_filtrar">
     <property name="geometry">
      <rect>
       <x>272</x>
       <y>30</y>
       <width>291</width>
       <height>22</height>
      </rect>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QListView" name="listView_submenu">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>60</y>
       <width>256</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_inserirsubmenu">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>30</y>
       <width>251</width>
       <height>22</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Inserir SubMenus</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Cardápio Montado</string>
    </attribute>
    <widget class="QTableView" name="tableView_submenuitens">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>751</width>
       <height>341</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_filtrarsubmenuitens">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>10</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_filtrarsubmenuitens">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>10</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Filtrar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_apagarsubmenuitens">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>40</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Apagar 
seleção</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_tamanho">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_tipo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
